src/fragment-components/a-fragment.tsx

```diff
@@ -116,6 +116,7 @@ export const componentInfo: ComponentInfo = {
 			inputs: (_) => '',
 			outputs: (_) => '',
 			imports: [],
+			otherImports: (_) => '',
```

Comment:
Do we really need this? It's optional, isn't it? And returning empty string does nothing anyway?